data/json/items/book/misc.json

```
@@ -1098,7 +1098,7 @@
       },
       {
         "id": "philosophy6",
-        "text": "A collection of texts and essays by Jacques Derrida.  It's pages loose and yellowed, you should probably handle it with care."
+        "text": "A collection of texts and essays by Jacques Derrida.  Its pages loose and yellowed, you should probably handle it with care."
```

Are loose.

---

Would probably swap that comma for a hyphen then
```suggestion
        "text": "A collection of texts and essays by Jacques Derrida.  Its pages are loose and yellowed - you should probably handle it with care."
```